Android.mk

```diff
@@ -0,0 +1,23 @@
+LOCAL_PATH:= $(call my-dir)
+
+include $(CLEAR_VARS)
+
+LOCAL_MODULE := Silence
+LOCAL_MODULE_TAGS := optional
+LOCAL_PACKAGE_NAME := Silence
+
+silence_root  := $(LOCAL_PATH)
+silence_build := $(silence_root)/build
+silence_apk   := build/outputs/apk/Silence-release-unsigned.apk
```

Comment:
```suggestion
silence_apk   := $(silence_build)/outputs/apk/Silence-release-unsigned.apk
```

Reply:
I doubt if It works. Because `silence_apk` should be referred relatively, but `silence_build` is an absolute path.